controllers/student.go

```diff
@@ -0,0 +1,127 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"kwoc-backend/middleware"
+	"kwoc-backend/utils"
+	"net/http"
+
+	"kwoc-backend/models"
+
+	"github.com/rs/zerolog/log"
+	"gorm.io/gorm"
+)
+
+type RegisterStudentReqFields struct {
+	Username string `json:"username"`
+	Name     string `json:"name"`
+	Email    string `json:"email"`
+	College  string `json:"college"`
+}
+
```

Comment:
Is this an exhaustive list of parameters required for registering a student?

Reply:
Yes.